generate_latest_sums.sh

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-set -eo pipefail
+set -o pipefail
```

Comment:
Why remove the `-e`?

Reply:
The availability check for openj9 on aarch64 below fails (It is not available right now). But we want the script to continue. This will also ensure that when Openj9 becomes available on aarch64 it will work with no code changes.

Reply:
OK thanks!